feature/gnmi/ate_tests/telemetry_basic_check_test/telemetry_basic_check_test.go

```diff
@@ -395,40 +395,36 @@ func TestComponentParent(t *testing.T) {
 
 	for _, tc := range cases {
 		t.Run(tc.desc, func(t *testing.T) {
-			t.Logf("Found card list for Component Type %v : %v", tc.componentType, cardList[tc.desc])
+			t.Logf("Found Component list for Type %v : %v", tc.componentType, cardList[tc.desc])
 			if len(cardList[tc.desc]) == 0 {
-				t.Fatalf("Get Card list for %q: got 0, want > 0", dut.Model())
+				t.Fatalf("Get Component list for %q: got 0, want > 0", dut.Model())
 			}
 			// Validate parent component
 			for _, card := range cardList[tc.desc] {
```

Comment:
Please rename "card" and "cardList" to "comp" and "compList" to align with your better logging here that these are generally components, not cards